lib/saml/user_attributes/ssoe.rb

```diff
@@ -50,6 +56,27 @@ def email
       end
 
       ### Identifiers
+      def uuid
+        raise Common::Exceptions::InvalidResource, @attributes unless idme_uuid || sec_id
+
+        return idme_uuid if idme_uuid
+
+        Digest::UUID.uuid_v3('sec-id', sec_id).tr('-', '')
```

Comment:
@annaswims I don't know.  My guess is V4, but even if they did use V3 there is still a chance for collision since the namespace would be different (although the chance is tiny).

In my [comment to Bill](https://github.com/department-of-veterans-affairs/vets-api/pull/4027#discussion_r400364760) earlier was trying to imply that I think the chance of a UUID collision is minimal compared to the possibility of a downstream process assuming that all user UUIDs are going to be 32 bytes.  However, if that's a bad assumption, we should remove the hashing process and just return the raw `sec_id`